src/main/java/bit/javaoop/Employee.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+public class Employee {
+    private double salary;
+    private String name;
+
+    public Employee(int salary, String name) {
+        this.salary = salary;
+        this.name = name;
+    }
+
+    public boolean isSatisfied() {
+        return this.salary > 10_000;
```

Comment:
_magic number_ to code smell ta liczba powinna zostać wyekstrahowana to stałej (alt + ctrl + C) o nazwie np. SATISFYING_SALARY zgodnie z konwencją